src/native/corehost/hostpolicy/deps_resolver.cpp

```diff
@@ -735,17 +735,23 @@ void deps_resolver_t::resolve_additional_deps(const arguments_t& args, const dep
 
     for (pal::string_t json_file : m_additional_deps_files)
     {
-        m_additional_deps.push_back(std::unique_ptr<deps_json_t>(
-            new deps_json_t(true, json_file, rid_fallback_graph)));
+        if (rid_fallback_graph != nullptr)
+        {
+            m_additional_deps.push_back(std::unique_ptr<deps_json_t>(
+                new deps_json_t(true, json_file, *rid_fallback_graph)));
```

Comment:
```suggestion
            m_additional_deps.push_back(std::make_unique<deps_json_t>(true, json_file, *rid_fallback_graph));
```

Reply:
I had originally used that, but that broke unix builds, since, unless I'm mistaken, `make_unique` is only in C++14.

Reply:
:(